tests/test_monte_carlo.c

```diff
@@ -0,0 +1,152 @@
+#include <monte_carlo.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void playAction(Board* board, Action* action) {
+  board->values[action->xPos * 3 + action->yPos] = action->player;
+}
+
+static bool isValidAction(Board* board, Action* action, int player) {
+  if (action->player == player &&
+      board->values[action->xPos * 3 + action->yPos] == 0) {
+    return 1;
+  }
+  return 0;
+}
+
+void getPossibleActions(Board board, Action possibleActions[]) {
+  int nActions = 0;
+  int player = 1;
+  for (int x = 0; x < 3; ++x) {
+    for (int y = 0; y < 3; ++y) {
+      if (board.values[x * 3 + y] == 0) {
+        for (int i = 0; i < board.nPlayers; ++i) {
+          possibleActions[nActions].player = player;
+          possibleActions[nActions].xPos = x;
+          possibleActions[nActions].yPos = y;
+          nActions++;
+          player = -player;
+        }
+      }
+    }
+  }
+}
+
+static int getNumPossibleActions(Board board) {
+  int nActions = 0;
+  for (int x = 0; x < 3; ++x) {
+    for (int y = 0; y < 3; ++y) {
+      if (board.values[x * 3 + y] == 0) {
+        nActions++;
+      }
+    }
+  }
+  return nActions * board.nPlayers;
+}
+
+static int getBoardSize() { return 9; }
+
+static bool isDone(Board* board) {
+  for (int i = 0; i < 9; ++i) {
+    if (board->values[i] == 0) {
+      return false;
+    }
+  }
+  return true;
+}
+
+/**
+ * If there is a line of X or O's
+ * 1 = draw
+ * 2 = win
+ * 3 = win with only 3 plays
+ */
+int getScore(Board* board, int player) {
+  // Check rows
+  for (int i = 0; i < 7; i += 3) {
```

Comment:
7 ?

Reply:
For the size of the board, I changed it to a define!